### PR TITLE
ensure absolute path plugins are hashed correctly

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: tests/utils/options-hash.js [hunk 1]
```diff
@@ -224,6 +224,40 @@ describe('optionsHash', function() {
...
+      expect(hash1).to.eql(hash1b);
+      expect(hash1).to.not.eql(hash1c);
+
+      project.version = '3.1.5';
+      project.writeSync();
+
+      const hash2 = optionsHash({ plugins: [project.root + '/' + project.name]}, console);
+      expect(hash1).to.eql(hash2);
+
+      require('hash-for-dep')._resetCache();
```
Comment (stefanpenner) [inline tests/utils/options-hash.js]: @rwjblue we could make this indirect via `optionsHash._resetCache()` instead...

**Comment by pzuraq** [comment]
Where are we on this?

**Comment by rwjblue** [comment]
I'd like to see it landed and released, but when I chatted with @stefanpenner about it he was concerned about performance implications. 

@stefanpenner - Do you think it would be OK to land and release this to prevent user trollage, and monitor our build performance benchmarks that are already in place for any issue after updating?

**Comment by rwjblue** [comment]
Thank you @stefanpenner! I've PR'ed `ember-cli-babel` to ensure it gets a patch release with this fix...

**Pull request merged**